cluster/ci/config/mirroring/origin_v3_11

```diff
@@ -5,6 +5,11 @@ registry.svc.ci.openshift.org/openshift/origin-v3.11:cli docker.io/openshift/ori
 registry.svc.ci.openshift.org/openshift/origin-v3.11:cluster-autoscaler docker.io/openshift/origin-cluster-autoscaler:v3.11 docker.io/openshift/origin-cluster-autoscaler:v3.11.0 docker.io/openshift/origin-cluster-autoscaler:latest
 registry.svc.ci.openshift.org/openshift/origin-v3.11:cluster-capacity docker.io/openshift/origin-cluster-capacity:v3.11 docker.io/openshift/origin-cluster-capacity:v3.11.0 docker.io/openshift/origin-cluster-capacity:latest
 registry.svc.ci.openshift.org/openshift/origin-v3.11:console docker.io/openshift/origin-console:v3.11 docker.io/openshift/origin-console:v3.11.0 docker.io/openshift/origin-console:latest
+registry.svc.ci.openshift.org/openshift/origin-v3.11:prometheus-operator docker.io/openshift/prometheus-operator:v3.11 docker.io/openshift/prometheus-operator:v3.11.0 docker.io/openshift/prometheus-operator:latest
```

Comment:
Can't set these until you have the jobs ready.  Generally this file is updated by running `make image-mirror-files` once you are publishing your branch jobs from ci-operator based jobs in prow.  If your image build jobs aren't in ci-operator form (they should be!) then this isn't necessary.

Reply:
If you need examples of adding new images, https://github.com/openshift/release/pull/1051 is pretty much the canonical example right now.